system_tasks.yml

```diff
@@ -15,6 +15,26 @@
     - python-dev
     - gdal-bin
     - unzip
+    - python-apt
+
+# By default, Piecewise uses EFF's LetsEncrypt CA for obtaining SSL certificates.
+# If you have obtained SSL certificates from another source, you should comment out the 
+# lines below, ending at the line starting with: `shell: "certbot ...` and then edit 
+# the Nginx default site to point at your certificates: /conf/etc/nginx/sites-available/default
+- apt_repository:
+    repo: deb http://ftp.debian.org/debian jessie-backports main
+    state: present
+- name: Update apt database to get jessie-backports
+  apt: update_cache=yes
+- name: Install EFF Certbot
+  become: yes
+  shell: "apt-get install certbot -t jessie-backports"
+- name: Generate Letsencrypt certs
+  shell: "certbot certonly --standalone -n -d piecewise.opentechinstitute.org -m critzo@opentechinstitute.org --agree-tos"
+
+- name: Enable nginx default site
+  file: src=/etc/nginx/sites-available/default dest=/etc/nginx/sites-enabled/default state=link 
```

Comment:
For all tasks, could we put attribute lines on separate lines, indented (here and elsewhere)? It's much more readable:
```
- name: Enable nginx default site
  file:
    src=/etc/nginx/sites-available/default
    dest=/etc/nginx/sites-enabled/default
    state=link
```